app/ui-react/syndesis/src/index.css

```diff
@@ -1,4 +1,5 @@
 body {
+  display: flex;
```

Comment:
I feel like this line should scare me a little bit, I dunno why.  Am suspicious.

Reply:
So I added this to fix an issue where the background color of the header doesn't fit the full width of the viewport on really small screen sizes. Can remove, but seems like we might want it until the issue is fixed on PF side.

Reply:
I'm personally a +1 for waiting for the official fix.

Reply:
Cool, I'm pretty sure is a fudge between pf3/pf4 - will look into it. Keep ya posted!

Reply:
Looks like the issue is that the user menu in the masthead makes the header too wide. We move the user menu (and other masthead icons/menus) into a kebab on mobile in pf4. You can see that general behavior here - https://pf4.patternfly.org/demos/Page/examples-full/?component=Page%20component%20default%20nav%20header%20example

If we change the user menu dropdown to an icon or something more narrow, the header won't overflow the x-axis of the page, and `display: flex` shouldn't be needed.